core/providers/anthropic/anthropic.go

```diff
@@ -480,6 +491,35 @@ func HandleAnthropicChatCompletionStreaming(
 			return
 		}
 
+		// Track last activity time for inactivity timeout detection
```

Comment:
@pjcdawkins can you have a common util for this? also lets add a field in the UI as well for stream activity timeout in network config